instrumentation/afl-compiler-rt.o.c

```diff
@@ -761,7 +796,12 @@ static void __afl_start_snapshots(void) {
     if (child_stopped && was_killed) {
 
       child_stopped = 0;
-      if (waitpid(child_pid, &status, 0) < 0) _exit(1);
+      if (waitpid(child_pid, &status, 0) < 0) {
+
+        write_error("child_stopped && was_killed");
```

Comment:
could we add "waitpid" to the description?

Reply:
there is already one with that text. I dont want duplicate messages

Reply:
Neither do I, so just _add_ that?

Reply:
this write error specifies a specific location. so it is unique. if you prefer a different message that is unique too - go for it. this one tells me where it goes wrong.

Reply:
ok, fine with me.